binstat/README.md

```diff
@@ -0,0 +1,180 @@
+# `binstat` - run-time & space efficient stats in bins
```

Comment:
I think it might be better binstat codes to be part of utils package instead of a root packages.

Reply:
@ramtinms thanks for the comment and sorry for not positioning it better the first time around!

I am in the process of moving it from `/binstat` to `/utils/binstat`. Interestingly, after moving it then the tests run later -- presumably at a more test crowded time because of running all the tests? -- and this generates a `WARNING: DATA RACE` which I've never seen before and I'm looking into :-)

Reply:
Looks like some values written atomically were not being read atomically (intentionally) during the dump function. But Golang's data race cares about that so I just started reading them atomically as well :-)

I pushed the update and now `binstat` is located at `utils/binstat/` :-)